fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialog.kt

```diff
@@ -49,9 +52,45 @@ fun TranslationOptionsDialog(
 ) {
     TranslationOptionsDialogHeader(onBackClicked)
 
+    val translationOptionsListState = remember {
+        translationOptionsList.toMutableStateList()
+    }
+
+    EnabledTranslationOptionsItems(
+        translationOptionsListState,
+    )
+
     LazyColumn {
-        items(translationOptionsList) { item: TranslationSwitchItem ->
-            TranslationOptions(translationSwitchItem = item)
+        items(translationOptionsListState) { item: TranslationSwitchItem ->
```

Comment:
Could you also go ahead and add logic for always showing the descriptions while in [landscape](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?node-id=7796%3A15675&mode=dev) too?

Reply:
In landscape it works the same like in portrait . I put a comment in Figma; I don't think the design is correct.
https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD?node-id=7796:15675&mode=dev#650459280

Reply:
Thanks for checking with the design team!